cnew.py

```diff
@@ -126,6 +126,25 @@ def formusnm(username):
     elif len(username) > 10:    return username[0:10]
     else:                       return username
 
+def check_socket(servaddr):
+    addr= [x.strip().strip("/") for x in servaddr.split(":")] #['ws',ip, port_no]
```

Comment:
That's an intelligent way of splitting the URI into protocol, address and port number, facilitating error correction to a greater detail. Great work, @gitarthasarma :star2:!